src/mumble/VersionCheck.cpp

```diff
@@ -72,8 +82,26 @@ VersionCheck::VersionCheck(bool autocheck, QObject *p, bool focus) : QObject(p)
 	url.setQuery(query);
 
 	WebFetch::fetch(QLatin1String("update"), url, this, SLOT(fetched(QByteArray,QUrl)));
+#else // When USE_APPIMAGE_UPDATER_BRIDGE is defined
+// USE_APPIMAGE_UPDATER_BRIDGE implies that we are on a linux machine 
```

Comment:
Is this enforced somewhere? Aka does the build system throw an error if I define `USE_APPIMAGE_UPDATER_BRIDGE` on another OS than Linux, will it throw an error?

Reply:
No, the build system will not throw an error when you define ```USE_APPIMAGE_UPDATER_BRIDGE``` on other OS than Linux.

Reply:
Then I'd say this should be changed. Otherwise I think this could lead to strange reports on builds failing with weird error messages just because people messed with the build configuration and enabled this one on a system it doesn't belong to.

Reply:
I've made the checking a little bit strict but I assume the developer does not pass ```APPIMAGE_UPDATER_BRIDGE_ENABLED``` config to qmake in a non-Linux machine.

Reply:
See https://github.com/mumble-voip/mumble/pull/3969/files#diff-5e71cd8a9cdd2b262ca0000d69aa3a6aR8, is this enough??

Reply:
I'd simply add an `else` block to the `unix` property and check if `APPIMAGE_UPDATER_BRIDGE_ENABLED` is set. And if it is, error out.

If this flag is only set by script, that's good. However we should be better safe than sorry. You never know where people dig for information on what flags are available (but potentially undocumented - sometimes for a good reason xD)